README.md

```diff
@@ -426,4 +426,4 @@ We welcome your contributions! Please have a read of
 
 [dev-ware]: https://github.com/webpack/webpack-dev-middleware#options
 [hot-client]: https://github.com/webpack-contrib/webpack-hot-client#options
-[https-opts]: https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options
\ No newline at end of file
+[https-opts]: https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options
```

Comment:
I am not sure where it got the latter change. Newline?

Reply:
yeah, no big deal that